docker/jupyterhub.sh

```diff
@@ -217,9 +217,10 @@ function set-default-configs() {
 function set-default-name-pattern() {
   if [ -z "${CLUSTER_NAME_PATTERN}" ];
   then
-    default_name_pattern="dataprochub-$( curl -s ${metadata_base_url}/instance/name -H "Metadata-Flavor: Google" )-{}"
+    default_name_pattern="dataprochub-{}"
```

Comment:
Any reasons for not keeping the Hub instance name in the cluster name? Makes it easier to identify which cluster belongs to which hub.

On the down side, it makes the name quite long (but we could replace dataprochub by hub or dphub maybe)

Reply:
Ack, I've changed the prefix to be 'hub-' instead and I'll keep the instance name for now.